azure/packages/azure-client/package.json

```diff
@@ -72,12 +72,12 @@
 		"axios": "^0.26.0"
 	},
 	"devDependencies": {
-		"@fluid-tools/build-cli": "^0.27.0",
+		"@fluid-tools/build-cli": "0.28.0-210888",
```

Comment:
Why isn't the version `^0.28.0`?

Reply:
Semver won't let that grab prereleases, IIRC. If we already had an official release of 0.28.0, then I agree with the version you suggested.

Reply:
I did a prerelease rather than a full release because it's moderately easier to "patch" than a full release, and my changes in particular may need revision. Had I done a full release, the version range here would indeed be `^0.28.0`.